### DISCUSSION TITLE
Need Rustdesk web version

### DISCUSSION THREAD

**Discussion by NeerajBisht** [rustdesk/rustdesk#4812] [General]
Hello, 
I'm currently working on implementing the web version of Flutter, but I'm facing some challenges when using Docker and command-line instructions. Are you in the process to updating the project? The web beta version appears to be functional, but when accessing the system remotely, the screen doesn't fit properly and displays scroll bars. Additionally, the remote screen moves along with the mouse. Could you please assist me with these issues? when we get the latest updated code for the same?

**Comment by rustdesk** [comment]
No plan yet, our limited resource does not allow us.

**Reply by NeerajBisht** [reply]
can i get something to get remote access system via web browser from Rustdesk?
